Source/GameBaseFramework/Public/Camera/GBFCameraMode.h

```diff
@@ -0,0 +1,203 @@
+#pragma once
+
+#include <CoreMinimal.h>
+#include <GameplayTagContainer.h>
+#include <UObject/Object.h>
+
+#include "GBFCameraMode.generated.h"
+
+class UGBFCameraComponent;
+class UCanvas;
+
+#define GBF_CAMERA_DEFAULT_FOV ( 80.0f )

```

Comment:
urk

I would replace with something like

```
namespace GBF
{
    namespace CameraMode
{
    static const float DefaultFov;
// etc...
}
}
```

and the matching initialization in the cpp file

Reply:
No need for the C++ init, if we want them const we need to init in header